templates/php.ini.j2

```diff
@@ -200,6 +200,12 @@ session.hash_function = 0
 session.hash_bits_per_character = 5
 session.use_strict_mode = 0
 
+redis.session.lock_retries = {{ php_redis_session_lock_retries }}
```

Comment:
What happens if we are not using redis for sessions?

Reply:
we would need to change the other variables along with these. The logic to support other ways to save sessions could be done afterwards

Reply:
We can add a boolean variable to control whether these vars are added or not in the file?